src/test/ui/asm/srcloc.stderr

```diff
@@ -0,0 +1,74 @@
+error: invalid instruction mnemonic 'invalid_instruction'
+  --> $DIR/srcloc.rs:11:15
+   |
+LL |         asm!("invalid_instruction");
+   |               ^
+   |
+note: instantiated into assembly here
+  --> <inline asm>:2:2
+   |
+LL |     invalid_instruction
+   |     ^^^^^^^^^^^^^^^^^^^
+
+error: invalid instruction mnemonic 'invalid_instruction'
+  --> $DIR/srcloc.rs:15:13
+   |
+LL |             invalid_instruction
+   |             ^
```

Comment:
It would be nice if this could point at the entire width of `invalid_instruction`. ([I saw the `FIXME`](https://github.com/rust-lang/rust/pull/72625/files#diff-6e23b9d7fff06f32f7b69cd2e63b0826R384-R387), which is why this isn't the case.)